### PR TITLE
Logo brand assets

### PR DESCRIPTION
**Pull request by fsansalvadore**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Quick links to brand assets by right clicking the Supabase logo.

## What is the new behavior?

Screenshot 2024-06-17 at 18 35 32

## Additional context

I preferred using the DropdownMenu rather than the ContextMenu component since it looks more professional if it always open in the same position.

### CODE REVIEW

File: apps/www/components/Nav/RightClickBrandLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+            className="hidden dark:block"
+            priority
+          />
+        </Link>
+      </DropdownMenuTrigger>
+      <DropdownMenuPortal container={triggerRef.current}>
+        <DropdownMenuContent ref={ref} align="start" side="bottom" className="mt-2 p-0 w-52">
+          {menuItems.map((section, sectionIdx) => (
+            <Fragment key={`cxtMenu-section-${sectionIdx}`}>
+              {sectionIdx !== 0 && <Divider />}
```
Comment (MildTomato) [inline apps/www/components/Nav/RightClickBrandLogo.tsx:70]: `<DropdownSeperator />`

File: apps/www/components/Nav/RightClickBrandLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+              <div className="p-1">
+                {section.map((menuItem) => (
+                  <DropdownMenuItem
+                    key={menuItem.label}
+                    className="group/menu-item w-full flex justify-between gap-2 items-center p-2"
+                  >
+                    <MenuItem
```
Comment (MildTomato) [inline apps/www/components/Nav/RightClickBrandLogo.tsx:77]: Confusing name for a local component, as there is a MenuItem component in www/components

File: apps/www/components/Nav/RightClickBrandLogo.tsx [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+  const Children = () => (
+    <>
+      {icon && <span className="text-foreground-lighter">{icon}</span>}
+      <span className="grow">{children}</span>
+      {type === 'clipboard' && (
+        <span className="relative w-3 opacity-0 flex items-center justify-end group-hover/menu-item:opacity-100">
+          <IconClipboard
+            className={cn(
+              'absolute right-0 h-3 transition-opacity opacity-0 duration-300',
```
Comment (MildTomato) [inline apps/www/components/Nav/RightClickBrandLogo.tsx:240]: bit weird these are absolute.

**Review by MildTomato — APPROVED** [review]
approved with comments.

also some deprecated components in use

File: apps/www/components/Nav/RightClickBrandLogo.tsx (hunk 1 above)
Comment (fsansalvadore) [reply apps/www/components/Nav/RightClickBrandLogo.tsx]: yeah better 👍

File: apps/www/components/Nav/RightClickBrandLogo.tsx [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+import Link from 'next/link'
+import { useClickAway } from 'react-use'
+import {
+  cn,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuPortal,
+  DropdownMenuTrigger,
+  IconCheck,
```
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/www/components/Nav/RightClickBrandLogo.tsx]: using hero icons now ✌️

File: apps/www/components/Nav/RightClickBrandLogo.tsx (hunk 2 above)
Comment (fsansalvadore) [reply apps/www/components/Nav/RightClickBrandLogo.tsx]: updated

**Pull request merged**